Detections/VectraAI/VectraDetect-HighSeverityDetection-by-Tactics.yaml

```diff
@@ -0,0 +1,75 @@
+id: 6414c848-ca8a-4fb1-9634-34e6d163f451
+name: Vectra AI Detect - Detections with High Severity 
+description: |
+  'Create an incident for high severity malicious behavior detected by Vectra AI (Threat score superior to 7.0). 
+  The Severity is a mapping with the Threat score assigned to a detection. It ranges between 0 and 10. 
+  The severity_threshold variable can be adjusted as desired.' 
+severity: High
+requiredDataConnectors:
+  - connectorId: AIVectraDetect
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CredentialAccess
+  - Discovery
+  - LateralMovement
+  - Collection
+  - CommandAndControl
+  - Exfiltration
+  - Impact
+relevantTechniques:
+query: |
+  // Edit this variable to only keep the tactics where an incident needs to be created (Defaults are: "COMMAND & CONTROL", "BOTNET ACTIVITY", "EXFILTRATION", "LATERAL MOVEMENT", "RECONNAISSANCE")   
+  let configured_tactics = dynamic(["COMMAND & CONTROL", "BOTNET ACTIVITY", "EXFILTRATION", "LATERAL MOVEMENT", "RECONNAISSANCE"]);
+  //default threshold is 7 (meaning a threat score of 70)
+  let severity_threshold = 7.0;
+  //Map by default to High Severity in Sentinel
+  let Severity = "High";
+  CommonSecurityLog
+  | where DeviceVendor == "Vectra Networks"
+  | where DeviceProduct == "X Series"
+  | where DeviceEventClassID != "campaigns" and DeviceEventClassID != "hsc" and DeviceEventClassID != "audit" and DeviceEventClassID != "health" and DeviceEventClassID != "asc"
+  | extend Category = extract("cat=(.+?);", 1, AdditionalExtensions) 
+  | project-rename threat_score = FlexNumber1
+  | project-rename certainty_score = FlexNumber2
+  | project-rename vectra_URL = DeviceCustomString4
+  | project-rename detection_name = DeviceEventClassID
+  | where todecimal(LogSeverity) >= severity_threshold
+  | extend Tactic = case( Category == "COMMAND & CONTROL", "CommandAndControl",
+                          Category == "BOTNET ACTIVITY" , "Impact",
+                          Category == "EXFILTRATION", "Exfiltration",
+                          Category == "LATERAL MOVEMENT", "LateralMovement",
+                          Category == "RECONNAISSANCE", "Discovery",
+                          "UNKNOWN")
+  | extend account = extract("account=(.+?);", 1, AdditionalExtensions)
+  | extend account_consolidated = iff(account matches regex ":", tostring(split(account,":")[1]) ,tostring(split(account,":")[0])) 
+  | extend source_entity = case( isnotempty(account_consolidated), account_consolidated,
+                                 isnotempty(SourceHostName), SourceHostName,
+                                 "UNKNWON") 
+  | where Category in (configured_tactics) 
+  | summarize arg_max(threat_score, *) by source_entity, Activity
+  | sort by TimeGenerated
+  | project source_entity, SourceHostName, SourceIP, account_consolidated, Tactic, Activity, LogSeverity, Severity, vectra_URL
```

Comment:
Do we want to map the different entity in the end using extend.
https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#old-entity-mapping-method

Also project TimeGenerated in the end and extend it as well.